apps/dao/lingui.config.ts

```diff
@@ -0,0 +1,6 @@
+import { config, defaultCatalog } from '../../lingui.config'
+
+export default {
```

Comment:
all apps export the same thing, can't we move defaultCatalog to the `config` object?

Reply:
Was debating, couldn't chose, figured maybe you want to extend later. Either way `defaultCatalog` is now moved inside the config directly.